.github/workflows/java-ci-sonarcloud.yml

```
@@ -1,15 +1,45 @@
+name: Java CI with Maven + SonarCloud
+
+on:
+  push:
+    branches: [ main, dev ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)